src/footprints/TRRS-PJ-320A-dual.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {

```

Comment:
First of all, thanks a lot for these additions... Gonna drop some quick general comments as far as I can understand stuff at a glance, and we can then discuss further after your thesis is submitted!